### PR TITLE
feat(e2e): implemented test to check download users link

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**
## Description

Implemented test to check download users link.

## Which issue(s) does this PR fix

- Fixes [RHIDP-3892](https://issues.redhat.com/browse/RHIDP-3892)

## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

### CODE REVIEW

File: e2e-tests/playwright/e2e/plugins/rbac/rbac.spec.ts [hunk 1]
```diff
@@ -150,6 +151,42 @@ test.describe.serial("Test RBAC plugin as an admin user", () => {
     await uiHelper.verifyCellsInTable(allCellsIdentifier);
   });
 
+  test("Should download the user list", async () => {
+    await page.locator('a:has-text("Download User List")').click();
+    const fileContent = await downloadAndReadFile(page);
+    const lines = fileContent.trim().split("\n");
+
+    const header = "userEntityRef,displayName,email,lastAuthTime";
```
Comment (gustavolira) [inline e2e-tests/playwright/e2e/plugins/rbac/rbac.spec.ts:159]: nitpick

can you please extract header validation to a specific function, for example:
```
function validateFileHeader(actualHeader: string, expectedHeader: string): void {
  if (actualHeader !== expectedHeader) {
    throw new Error(`Header mismatch: expected "${expectedHeader}" but found "${actualHeader}"`);
  }
```

and them you can call :

```
const lines = fileContent.trim().split("\n");
const expectedHeader = "userEntityRef,displayName,email,lastAuthTime";
validateFileHeader(lines[0], expectedHeader);
```

**Review by gustavolira — APPROVED** [review]

**Comment by AndrienkoAleksandr** [comment]
I didn’t change anything; I just rebased the PR.

**Review by PatAKnight — APPROVED** [review]
@gustavolira has already approved once and tests are passing

/lgtm

**Pull request merged**